test-spicy-features

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env bash
+
```

Comment:
This file is great but we should include `set -euxo pipefail` here. I had trouble with `-o pipefail` and the `cat` on line 28, we might need to add a `|| true` somewhere there.

Reply:
Done, works for me (after some other changes, too)